openverse_catalog/dags/providers/provider_api_scripts/inaturalist.py

```diff
@@ -72,38 +70,28 @@ def get_response_json(self, query_params: Dict):
         """
         Call the SQL to pull json from Postgres, where the raw data has been loaded.
         """
-        offset_num = str(query_params["offset_num"])
-        sql_string = self.sql_template.replace("offset_num", offset_num)
-        return self.pg.get_records(sql_string)
+        sql_string = self.sql_template.format(
+            batch_limit=self.batch_limit, offset_num=query_params["offset_num"]
+        )
+        sql_result = self.pg.get_records(sql_string)
+        # Postgres returns a a list of tuples, even if it's one tuple with one item.
+        return sql_result[0][0]
```

Comment:
This will only get the first item of the first tuple, and drop any of the rest of the records. Is that intended? To replicate what was happening in `get_batch_data`, this may need to be `return [dict(row[0]) for row in sql_result]

Reply:
This is intended! :) I changed [the sql](https://github.com/WordPress/openverse-catalog/pull/549/files#diff-32ee65dc27156dd551c587cbdd63225f7b6e414cc6fadf926a179c94618c5a0c), because the list parsing for tags was still broken and I dunno, I can definitely change it back if you think there will be a performance hit, but it feels much cleaner and simpler to me this way.

Reply:
I tried with the full dataset, and did not wait for `pull_data` to finish on my little mac, but:

```
[2022-07-29, 20:57:27 UTC] {python.py:173} INFO - Done. Returned value was: 
*Provider*: `inaturalist`
*Date range*: _all_
*Duration of data pull task*: 7 mins, 31 secs
*Number of records upserted per media type*:
  - `image`: 627,525 _(2,475 duplicate foreign IDs)_
```

Reply:
Oh whoops haha, okay ignore me then! 😄